app/components/support_interface/application_summary_component.rb

```diff
@@ -58,11 +58,17 @@ def submitted_row
       if submitted?
         {
           key: 'Submitted',
-          value: submitted_at.to_s(:govuk_date_and_time),
+          value: "#{submitted_at.to_s(:govuk_date_and_time)} #{eligible_support_period}".html_safe,
         }
       end
     end
 
+    def eligible_support_period
+      time_period = submitted_at.to_date.business_days_until(Time.zone.now)
+
+      time_period <= 5 ? '<b style="color:green;">LESS THAN 5 DAYS AGO</b>' : '<b style="color:red;">OVER 5 DAYS AGO</b>'
```

Comment:
We should probably use govuk tag class for this? and upcase in CSS. Defer to @paulrobertlloyd

Reply:
Yup; think you can use the `govuk_tag` helper, with capitalised text as @duncanjbrown suggests. Something like (off the top of my head):

```
<%= govuk_tag(text: 'Less than 5 days ago', colour: 'green') %>
```